spec/models/dialog_spec.rb

```diff
@@ -61,13 +62,13 @@
 
   context "#create" do
     it "validates_presence_of name" do
-      expect { FactoryGirl.create(:dialog) }.to raise_error
+      expect { FactoryGirl.create(:dialog) }.to raise_error(ActiveRecord::RecordInvalid, /Label can't be blank/)
```

Comment:
I agree, but for another PR.  Base factories that are broken make no sense to me.